core/runtime/src/main/java/io/quarkus/runtime/LocalesBuildTimeConfig.java

```diff
@@ -51,6 +51,6 @@ public class LocalesBuildTimeConfig {
      * Defaults to the JVM's default locale if not set. Starting with GraalVM for JDK 24, it defaults to {@code en-US}
      * for native executables.
```

Comment:
I think this probably should be in `defaultValueDocumentation`.

Reply:
But can we have a `defaultValudeDocumentation` without a `defaultValue`?

Reply:
Fixed.

Reply:
@zakkak sorry, missed your question: yes you can, we use that a lot.